### ISSUE TITLE
NPM publisj from CI environment

### ISSUE THREAD

**Issue by mrchief** [webpro/release-it#357]
How does this tool release to NPM from a CI env like travis? I don't see any optoin to pass the npm token for publishing.

**Comment by webpro** [comment]
If you mean the One-Time Password: you can pass it with `--npm.otp=123456`. But this token expires quickly, and from a security perspective it should be a manual/interactive action. So publishing to npm using an OTP doesn't fit well in CI.

**Comment by mrchief** [comment]
Not the otp, just the auth token. How would you release it from a ci build? Say like travis.

**Comment by webpro** [comment]
Yes, looks like there's no way to provide it to npm-publish directly.

**Comment by webpro** [comment]
It's actually possible to publish npm packages with release-it from a CI/CD environment: https://github.com/release-it/release-it/blob/master/docs/ci.md
